### PR TITLE
Add tvOS support

### PR DESCRIPTION
**Pull request by warmenhoven**

### CODE REVIEW

File: src/PathUtils.cpp [hunk 1]
```diff
@@ -57,9 +57,13 @@ fs::path PathUtils::GetCachePath()
 
 fs::path PathUtils::GetRoamingDataPath()
 {
+#if TARGET_OS_TV
+    return GetCachePath();
```
Comment (jpd002) [inline src/PathUtils.cpp:61]: nit: Can you use a tab to indent this line? Thanks!

Comment (jpd002) [inline src/PathUtils.cpp:61]:
```suggestion
	return GetCachePath();
```

Comment (warmenhoven) [reply src/PathUtils.cpp]: Yes, absolutely. I pushed an updated PR at the same time that the suggested change below came in. Thanks!

Comment (jpd002) [reply src/PathUtils.cpp]: Great! Thanks again!

**Pull request merged**